### PR TITLE
LPS-156594 Tests Automation Add a Results Bar to the Categories Filter in Content Dashboard

### PR DESCRIPTION
**Pull request by rafaellajordao**
JIRA Ticket:
[LPS-155288](https://issues.liferay.com/browse/LPS-155288)
↳ [LPS-156594](https://issues.liferay.com/browse/LPS-156594)

Motivation:
Add a Results Bar to the Categories Filter in Content Dashboard

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 1]
```diff
@@ -818,6 +818,313 @@ definition {
...
+				locator1 = "Treeview#FILTER_BY_RESULTS_BAR_MESSAGE",
+				value1 = "3 ");
+
+			AssertTextPresent(
+				locator1 = "Treeview#FILTER_BY_RESULTS_BAR_MESSAGE",
+				value1 = "Items Selected");
+		}
+
+		task ("Assert 'Clear All' disapears after clicking it") {
+			ClickNoError(
```
Comment (Luizcarlosqueiroz) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:887]: Hey! What do you think about creating a reusable macro for `ClickNoError` and `AssertElementNotPresent` ?

**Comment by rafaellajordao** [comment]
ci:test:tango

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase (hunk 1 above)
Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase]: Hello Luiz! great idea! :)

I created the new macro, could you review it again? Thanks!

**Comment by rafaellajordao** [comment]
> ###  ci:test:tango - 163 out of 175 jobs passed in 1 hour 55 minutes
> Click here for more details.

these failures have nothing to do with the test correction :)

**Review by Luizcarlosqueiroz — APPROVED** [review]

**Comment by Luizcarlosqueiroz** [comment]
ci:forward

**Pull request closed without merging**